src/views/GovernanceTaskList/index.test.tsx

```diff
@@ -392,5 +393,82 @@ describe('The Goveranance Task List', () => {
           .text()
       ).toEqual('Completed');
     });
+
+    it('renders proper buttons for BIZ_CASE_FINAL_NEEDED', async () => {
+      const store = mockStore({
```

Comment:
It might be interesting to see if we could mock the return value of `useSelector` instead of mocking the store. Would that let us `shallow` render instead of `mount` our components? Something to think about for the future, but def outside the scope of this PR.